src/notifier/notion.py

```
@@ -316,3 +319,148 @@ def send_to_notion(
+    if block_type == "numbered_list_item":
+        return f"1. {text}"
+    if block_type == "divider":
+        return "---"
+    if block_type == "table_row":
+        cells = content.get("cells", [])
+        return "| " + " | ".join(_rich_text_to_str(cell) for cell in cells) + " |"
+    return text
+
+
+def _fetch_page_text(notion: Client, page_id: str) -> str:
+    """ページ全ブロックをテキストに変換して返す（ページネーション対応）。"""
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)